.cicd/platforms/pinned/ubuntu-18.04-pinned.dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apt-get update && \
     libtasn1-dev libnss3-dev iproute2 expect gawk socat python3-pip libseccomp-dev uuid-dev && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
+# install request and requests_unixsocket module
+RUN pip3 install requests && pip3 install requests_unixsocket
```

Comment:
This can just be
`pip3 install requests requests_unixsocket`

Reply:
Done.